tools/sidekick/config.js

```diff
@@ -213,7 +213,8 @@
     if (!webPath) {
       return;
     }
-    if (!webPath.split('/').pop().includes('.')) {
+    const file = webPath.split('/').pop();
+    if (file && !file.includes('.')) {
```

Comment:
I don't see the difference here: to me, both variants get the last `/` segment and add `.html` if that segment has no extension. The new version additionally checks whether the last segment is non-empty (so it wouldn't add `.html` to e.g. `/my-path//`).

Since you only want to check a `.` in the *last* segment, what about:
```
const pathSep = webPath.lastIndexOf('/');
if (pathSep !== -1 && webPath.indexOf('.', pathSep + 1) === -1( {
   // no extension
  sk.status.webPath = `${webPath}.html`;
}
```
This would avoid creating the array.

Reply:
Ah, yes, now I see the difference 🙂

Reply:
Changed the suggested code to include the case where the last segment is empty!